back/internal/usersvc/internal/userdb/qryuser.go

```diff
@@ -81,17 +93,15 @@ func (s *UserDB) deleteUser(ctx cx, sid string) error {
 }
 
 func (s *UserDB) findUsers(ctx cx, x *pb.FndUsersReq, y *pb.UsersResp) error {
-	selStmt, err := s.db.Prepare("SELECT user_id, username, email, email_hold, altmail, altmail_hold, full_name, avatar, last_login, created_at, updated_at, deleted_at, blocked_at FROM user WHERE email IN ? OR email_hold = ? OR altmail in ? OR altmail_hold = ? LIMIT ? OFFSET ?")
+	selStmt, err := s.db.Prepare("SELECT user_id, username, email, email_hold, altmail, altmail_hold, full_name, avatar, last_login, created_at, updated_at, deleted_at, blocked_at FROM user WHERE email = ? AND email_hold = ? LIMIT ? OFFSET ?")
```

Comment:
We need to discuss how email_hold and altmail_hold are meant to be used.

Reply:
I agree. Just an idea, but I wonder if it would be good to switch to front end development. That way I can gain an understanding of what sorts of queries are needed to do what is needed to be done.